### DISCUSSION TITLE
Derivatives of the difference between two smooths

### DISCUSSION THREAD

**Discussion by BenFN121** [gavinsimpson/gratia#263] [Ideas]
Hi Gavin,

Is there any way that the capacity to apply the gratia::derivatives() to a difference_smooth created from gratia::difference_smooths() could be incorporated?

Many thanks,
Ben

**Comment by gavinsimpson** [comment]
I don't see any reason why not, in principle. I'll add this as a feature request.

**Reply by gavinsimpson** [reply]
See #264 and follow-up there.
